docs/_config.yml

```diff
@@ -1,94 +1,103 @@
-    name: C#/.NET
-  - value: http://dev.solace.com/get-started/java-tutorials/
-    name: Java
-  - value: http://dev.solace.com/get-started/javascript-tutorials/
-    name: JavaScript
-  - value: http://dev.solace.com/get-started/node-js-tutorials/
-    name: Node.js
-open-apis-dev:
-  - value: http://dev.solace.com/get-started/amqp-tutorials-jms-11/
-    name: AMQP - JMS 1.1
-  - value: http://dev.solace.com/get-started/amqp-tutorials-jms-2/
-    name: AMQP - JMS 2.0
-  - value: http://dev.solace.com/get-started/amqp-tutorials-node-js/
-    name: AMQP - Node.js
-  - value: http://dev.solace.com/get-started/mqtt-tutorials/
-    name: MQTT - Eclipse Paho Java
-  - value: http://dev.solace.com/get-started/amqp-tutorials-jms-11/
-    name: JMS 1.1 - Apache Qpid
-  - value: http://dev.solace.com/get-started/amqp-tutorials-jms-2/
-    name: JMS 2.0 - Apache Qpid
-  - value: http://dev.solace.com/get-started/jms-tutorials/
-    name: JMS 1.1 - Solace
-  - value: http://dev.solace.com/get-started/openmama-tutorials/
-    name: OpenMAMA
-  - value: http://dev.solace.com/get-started/rest-tutorials/
-    name: REST Messaging
-other-environments-dev:
-  - value: http://dev.solace.com/get-started/pcf-tutorials/
-    name: Pivotal Cloud Foundry
-
-solace-apis-cloud:
-  - value: http://cloud.solace.com/samples/solace-samples-c/
-    name: C
-  - value: http://cloud.solace.com/samples/solace-samples-dotnet/
-    name: C#/.NET
-  - value: http://cloud.solace.com/samples/solace-samples-java/
-    name: Java
-  - value: http://cloud.solace.com/samples/solace-samples-javascript/
-    name: JavaScript
-  - value: http://cloud.solace.com/samples/solace-samples-nodejs/
-    name: Node.js
-open-apis-cloud:
-  - value: http://cloud.solace.com/samples/solace-samples-amqp-qpid-jms1/
-    name: AMQP - JMS 1.1
-  - value: http://cloud.solace.com/samples/solace-samples-amqp-qpid-jms2/
-    name: AMQP - JMS 2.0
-  - value: http://cloud.solace.com/samples/solace-samples-amqp-nodejs/
-    name: AMQP - Node.js
-  - value: http://cloud.solace.com/samples/solace-samples-mqtt/
-    name: MQTT - Eclipse Paho Java
-  - value: http://cloud.solace.com/samples/solace-samples-amqp-qpid-jms1/
-    name: JMS 1.1 - Apache Qpid
-  - value: http://cloud.solace.com/samples/solace-samples-amqp-qpid-jms2/
-    name: JMS 2.0 - Apache Qpid
-  - value: http://cloud.solace.com/samples/solace-samples-jms/
-    name: JMS 1.1 - Solace
-  - value: http://cloud.solace.com/samples/solace-samples-openmama/
-    name: OpenMAMA
-  - value: http://cloud.solace.com/samples/solace-samples-rest-messaging/
-    name: REST Messaging
-
-#----------------------------------------------------
-# Generally don't modify items below this point
-url: "solacesamples.github.io" # the base hostname & protocol for your site
-
-# Google services
-google_analytics: UA-3921398-7
-
-# Build settings
-markdown: kramdown
-
-collections:
-  tutorials:
-    output: true
-    permalink: /:path/
+# Site settings

```

Comment:
Something is wrong with your diff on this file. Looks like you've converted the line endings or something. All lines are flagged as differed. Please investigate.